docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -0,0 +1,69 @@
+# Quality guidelines
+
+For the quick guidelines for the actual content (descriptions, screenshots), see the Appstream [docs](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html).
+
+The following guidelines are optional, but recommended. We will likely promote apps that follow them in certain places.
+
+## General
+
+### No trademark violations
+
+Your app must not violate any trademarks. This includes the name, icon, and summary.
+
+### Managed by app author
+
+We recommend to verify your app on Flathub, but it’s not required.
+
+## App Icon
+
+### Icon size
+
+The icon must be at least 256x256px and be in PNG or SVG format.
```

Comment:
minimum to pass the checks is 128x128

Reply:
```suggestion
The icon should be at least 256x256px and be in PNG or SVG format. The minimum accepted size on flathub is 128x128.
```

Reply:
This isn't very clear, it's either SVG (in which case the size doesn't matter) or PNG (in which case we want >256+).

```suggestion
The icon must be SVG, or a PNG with a size greater or equal to 256x256px.
```

> minimum to pass the checks is 128x128

This page isn't about passing the check, it's about high quality app listings (for which there's a higher bar).

Reply:
yeah but then it is confusing and you get submission where the review get second guessed by imprecise documentation.

Reply:
These rules are not for Flathub review, but only for high quality app listings. We should probably explain that somewhere on the page, but not sure we want to repeat all the minimum requirements for being allowed on Flathub at all on this page as well.

Reply:
pieces of information spread out all across the internet are what make things harder for contributors. A GOOD documentation do not hesitate to repeat important information. This is the flathub documentation for submissions so this is really relevant.

Reply:
> This page isn't about passing the check, it's about high quality app listings (for which there's a higher bar).

For now, I think in the long run, this will extend to general (e.g. not only listing) quality

Reply:
Realistically, if we want to mention the minimum size, we should link to the place where it's documented - otherwise, we need to update stuff in too many places.